### PR TITLE
[NFC][VP] Reduce parameters in `LoopVectorizePass::runImpl`

### PR DESCRIPTION
**Pull request by sunshaoce**
It seems that the parameters can be passed through the class members.

### CODE REVIEW

File: llvm/include/llvm/Transforms/Vectorize/LoopVectorize.h [hunk 1]
```diff
@@ -188,13 +188,7 @@ struct LoopVectorizePass : public PassInfoMixin<LoopVectorizePass> {
...
 
   // Shim for old PM.
-  LoopVectorizeResult runImpl(Function &F, ScalarEvolution &SE_, LoopInfo &LI_,
-                              TargetTransformInfo &TTI_, DominatorTree &DT_,
-                              BlockFrequencyInfo *BFI_, TargetLibraryInfo *TLI_,
-                              DemandedBits &DB_, AssumptionCache &AC_,
-                              LoopAccessInfoManager &LAIs_,
-                              OptimizationRemarkEmitter &ORE_,
-                              ProfileSummaryInfo *PSI_);
+  LoopVectorizeResult runImpl(Function &F);
```
Comment (nikic) [inline llvm/include/llvm/Transforms/Vectorize/LoopVectorize.h:191]: We could merge these entirely -- as the comment says, this was a shim for the old PM that is no longer needed.

**Review by nikic — APPROVED** [review]
LGTM, but note the formatting failure.

File: llvm/include/llvm/Transforms/Vectorize/LoopVectorize.h (hunk 1 above)
Comment (fhahn) [reply llvm/include/llvm/Transforms/Vectorize/LoopVectorize.h:191]: +1 for merging those

**Review by fhahn — APPROVED** [review]
LGTM ,thanks!

**Comment by sunshaoce** [comment]
I changed the indentation of `LoopVectorizePass::run` to two spaces.

**Pull request merged**